sunbird_service_api_test/src/test/resources/templates/user/create/testCreateUserFailureWithInvalidChannel/request.json

```diff
@@ -3,8 +3,8 @@
         "firstName": "ft_testName",
         "lastName": "test",
         "password": "password",
-        "userName": "citrus:concat('ft_username',citrus:randomString(10, MIXED, true))",
-        "email": "citrus:randomString(10, MIXED, true)@gmail.com",
+        "userName": "citrus:concat('ft_username',citrus:randomString(30, MIXED, true))",
```

Comment:
Are we using "FT_" or "ft_" as a convention?

Reply:
we are using ft_ one